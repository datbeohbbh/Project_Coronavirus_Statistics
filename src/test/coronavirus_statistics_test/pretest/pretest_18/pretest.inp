37 21
q 1 36 118
q 1 12 130
q 1 17 127
q 1 8 383
q 2 17 30
? 7
? 7
q 1 9 312
? 4 5
q 1 34 174
? 4 1
q 3 36 70
? 4 1
? 7
q 2 17 89
? 7
? 4 3
? 7
? 4 5
? 4 5
? 4 3