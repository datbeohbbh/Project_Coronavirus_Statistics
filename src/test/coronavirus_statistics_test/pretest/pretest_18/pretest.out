Query ? 7 :
Currently, no deaths have been recorded

Query ? 7 :
Currently, no deaths have been recorded

Query ? 4 5 :
The answer is the city with the index 8 
- Information of the city 8 -
-> Number of current patients : 383
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 17 
- Information of the city 17 -
-> Number of current patients : 97
-> Has recovered : 30
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 36 
- Information of the city 36 -
-> Number of current patients : 48
-> Has recovered : 0
-> Was dead : 70

Query ? 7 :
The city with the highest death is city with the index 36
Quantity: 70 people

Query ? 7 :
The city with the highest death is city with the index 36
Quantity: 70 people

Query ? 4 3 :
The answer is the city with the index 12 
- Information of the city 12 -
-> Number of current patients : 130
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 36
Quantity: 70 people

Query ? 4 5 :
The answer is the city with the index 9 
- Information of the city 9 -
-> Number of current patients : 312
-> Has recovered : 0
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 9 
- Information of the city 9 -
-> Number of current patients : 312
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 12 
- Information of the city 12 -
-> Number of current patients : 130
-> Has recovered : 0
-> Was dead : 0

